### ISSUE TITLE
ValueError

### ISSUE THREAD

**Issue by trishav96** [bxshi/ProjE#10]
I tried executing the softmax version of projE using python 3 and tensorflow 1.1.0
I get this value error. How do I fix this?
![Screenshot (1340)](https://user-images.githubusercontent.com/19915352/55755583-38279d00-5a04-11e9-8f6e-89f15eb11eb6.png)
I'm using the downloaded data as suggested in the readme (FB15k and WN18)

**Comment by bxshi** [comment]
I haven't experienced this issue before. I think this might due to an input data error? Seems the shape of your data does not match the placeholder.

**Comment by trishav96** [comment]
I downloaded the data from the link given.
